blackbox/docs/sql/subselects.txt

```diff
@@ -36,6 +36,21 @@ Multiple levels of sub selects are supported::
     +--------------+-------------+
     SELECT 5 rows in set (... sec)
 
+Joins in the sub selects are supported:
+
+    cr> select article, color from (
+    ...   select articles.name as article, colors.name as color, price
+    ...   from articles, colors
+    ...   where price > 5000.0) as t
+    ... where color ='Gold'
+    ... order by price desc;
+    +------------------------------+-------+
+    | article                      | color |
+    +------------------------------+-------+
+    | Starship Titanic             | Gold  |
+    | Infinite Improbability Drive | Gold  |
+    +------------------------------+-------+
+    SELECT 2 rows in set (... sec)
```

Comment:
adapt limitations section below and remove the "simple" line.  pls go over the doc here and avoid using the word "simple" since it odes not specify the characteristics of a subselect